mkt/prices/models.py

```diff
@@ -347,7 +347,13 @@ def create_addon_purchase(sender, instance, **kw):
                      c=instance))
         return
 
-    if instance.type == mkt.CONTRIB_PURCHASE:
+    print
+    print dir(instance.price_tier)
+    print instance.price_tier.price
+    print instance.price_tier.name
+
+    if instance.type == mkt.CONTRIB_PURCHASE \
+            and not instance.price_tier.name == '0':
```

Comment:
Yeah this is wrong at the moment. It's busted at this point as the user object is missing. I'm debugging that now. I'm expecting this code to run (and the added conditional will be removed).